portal/models/user.py

```diff
@@ -384,8 +384,9 @@ def all_consents(self):
     def valid_consents(self):
         """Access to consents that have neither been deleted or expired"""
         now = datetime.utcnow()
-        return self._consents.filter(
-            text("expires>:now and deleted_id is null")).params(now=now)
+        return [
```

Comment:
Why not return a filtered `Query` instead of a list? Usually I would expect the lazy evaluation would be better